docs/docs/cmd/spo/listitem/listitem-batch-add.mdx

```diff
@@ -54,6 +57,21 @@ Add a batch of items to a list retrieved by title in a specific site
 m365 spo listitem batch add --filePath "C:\Path\To\Csv\CsvFile.csv" --webUrl https://contoso.sharepoint.com/sites/project-x --listTitle "Demo List"
 ```
 
+Add a batch of items to a list retrieved by title in a specific site using csv content
+
+```sh
+$obj = [PSCustomObject]@{
+  Title = "Item A"
+  NumberField = 10
+}
```

Comment:
For the example, let's use a scenario where we use an object array instead of a singular object.